zeroconf.py

```diff
@@ -1949,17 +1959,22 @@ def create_sockets(
     interfaces = normalize_interface_choice(interfaces, ip_version)
 
     respond_sockets = []
+    respond_socket_options = []
```

Comment:
This seems wrong. You probably want to make this equal to `self._respond_socket_options`, otherwise on the level higher you will completely discard the previous value, when assigning the result of this function to `self._respond_socket_options`.

Reply:
The intent of the original author is that this function is called during initialization just once. So yes, it is intended to provide the full set of sockets that are in use at the time it is called.  If we proceed with the library, we can investigate making it aware of changing network interfaces such that this function is called when interfaces change.  At the moment this is only called from zeroconf.__init__